plugins/otp.c

```diff
@@ -700,8 +700,10 @@ static int parse_secret(const sasl_utils_t *utils,
 	    return SASL_FAIL;
 	}
 	
-	sscanf(secret, "%s\t%04d\t%s\t%s\t%020ld",
-	       alg, seq, seed, buf, timeout);
+	long long tmptimeout;
```

Comment:
Style nit - declaring variables in the middle of code is sloppy. At least, these lines should be enclosed as a block in { } brackets.